examples/gnn_node.py

```diff
@@ -200,13 +135,10 @@ def test(loader: NeighborLoader) -> np.ndarray:
     for batch in tqdm(loader):
         batch = batch.to(device)
         pred = model(
-            batch.tf_dict,
-            batch.edge_index_dict,
-            batch[entity_table].seed_time,
-            batch.time_dict,
-            batch.batch_dict,
-            batch.num_sampled_nodes_dict,
-            batch.num_sampled_edges_dict,
+            batch,
+            task.entity_table,
+            clamp_min=clamp_min,
+            clamp_max=clamp_max,
```

Comment:
Would make more sense to just clamp outside the model, would you agree?

Reply:
Makes sense.